documents/decorators.py

```diff
@@ -0,0 +1,19 @@
+from flask import jsonify, request
+import documents.state_machine as state_machine
+
+def auth(func):
+    def inner():
+        auth_header = request.headers.get('Authorization')
+        payload = state_machine.token_valid_check(auth_header)
+        return func(payload)
+    return inner
+
+def utils(func):
```

Comment:
I asked for the file containing common decorators to be called 'utils', not the decorator by itself :(

Reply:
rename function according to what it does